include/tunnler/tunnler.h

```diff
@@ -6,10 +6,14 @@
 #pragma once
 
 #include <array>
+#include <cstdint>
 #include <vector>
 
 using MacAddress = std::array<uint8_t, 6>;
 
+/// A special MAC address that tells the room we're joining to assign us a MAC address automatically.
+const MacAddress NoPreferredMac = { 0, 0, 0, 0, 0, 0 };
```

Comment:
Maybe just use the broadcast mac for that, since this should be the only mac address a client shouldn't have. 0:0:0:0:0:0 is as far as i know a legit mac address.